script.js

```diff
@@ -0,0 +1,78 @@
+const sumFix = 0.1 + 0.2;
+console.log(`Result - ${sumFix.toFixed(1)}`);
+
+const login = prompt("Введіть логін");
+const password = prompt("Введіть пароль");
+
+const corectLogin = "admin";
+const corectPassword = "12pass33210";
+
+if (login !== corectLogin && password !== corectPassword) {
+  console.log("Відмовлено в авторизації");
+} else if (login !== corectLogin || password !== corectPassword) {
+  console.log("Помилка в логіні або паролі");
+} else {
+  console.log("Вдала авторизація !");
+}
+
+const userFunds = prompt("Вкажіть ваш баланс!");
+
+const wmPrice = 21;
+const carrotPrice = 8;
+const potatoPrice = 12;
+
+console.log(`Асортимент :
+Кавун - ${wmPrice} гривня
+Морква - ${carrotPrice} гривень
+Картопля - ${potatoPrice} гривнень`);
+console.log(`Ваш баланс - ${userFunds} гривень`);
+
+if (userFunds >= Math.min(wmPrice, carrotPrice, potatoPrice)) {
+  const selectedItem = prompt("Оберіть продукт");
+  let itemsAmount = 0;
+  let fundsRest = 0;
+  switch (selectedItem) {
```

Comment:
What if the user will enter the product in lowercase?
As an option, you can add `.toLowerCase()` for such cases